pyodide-build/pyodide_build/cli/build.py

```diff
@@ -18,11 +20,30 @@
 )
 
 
+def convert_exports(exports: str) -> _BuildSpecExports | list[str]:
+    if "," in exports:
+        return [x.strip() for x in exports.split(",") if x.strip()]
+    if exports == "pyinit":
+        return "pyinit"
+    if exports == "requested":
+        return "requested"
+    if exports == "whole_archive":
+        return "whole_archive"
```

Comment:
Maybe:

```py
from typing import get_args
possible_exports = get_args(_BuildSpecExports)
if exports in possible_exports:
    return exports
```

Reply:
I think this pattern doesn't convince mypy? You could say `return cast(_BuildSpecExports, exports)`.

Reply:
Thanks for the input @henryiii and @ryanking13!